src/develop/pixelpipe_hb.c

```diff
@@ -2708,8 +2708,12 @@ gboolean dt_dev_pixelpipe_process(
                "[opencl] frequent opencl errors encountered; disabling"
                " opencl for this session!\n");
       dt_control_log
-        (_("darktable discovered problems with your OpenCL setup;"
-           " disabling OpenCL for this session!"));
+        (_("OpenCL errors encountered; disabling OpenCL for this session!"
+           " likely problems:\n"
+           "  - OpenCL out of resources due to preference settings. please try with defaults\n"
```

Comment:
add `,` at end of line. Change `.` on the 2 next lines by `,`.

Reply:
ok and done